packages/panda-san/src/navbar/navbar.scss

```diff
@@ -0,0 +1,107 @@
+.navbar {
+  align-items: center;
+  background-color: $navbar;
+  box-sizing: border-box;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  left: 0;
+  position: absolute;
+  width: 100vw;
```

Comment:
We should probably do `%` so we can nest this in smaller components if you want a `navbar` that isn't the `header`.